lesson-02/hw2/example-05.js

```diff
@@ -0,0 +1,19 @@
+const checkForSpam = function(message) {
+  let res = false;
```

Comment:
String теж містиль метод includes(), тому не потрібно виконувати операцію message.split(' ').
В результаті функція може виглядати ось так:
```suggestion
 const checkForSpam = function(message) {
   if (message.includes('spam') || message.includes('sale')) {
    return true;
  }
  return false;
 }
```